src/app/dadoPublicos/DadosPublicos.tsx

```
@@ -1,43 +1,77 @@
-		);
-	}, [selectedMunicipio]);
-
+    const router = useRouter();
+    const path = usePathname();
+    const searchParams = useSearchParams(); 
+    const painel = Number(searchParams.get('painel'));
+    const [activeTabIndex, setActiveTabIndex] = useState(painel || 0);
+    const [activeTitleTabIndex, setActiveTitleTabIndex] = useState(0);
+    const [scoreCardData, setScoreCardData] = useState([]);
+    const [selectedMunicipio, setSelectedMunicipio] = useState("João Pessoa - PB"); 
+    useEffect(() =>{ setActiveTabIndex(painel || 0)}, [path]);
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: painel

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)